task.sql

```diff
@@ -2,143 +2,13 @@ create database ShopDB;
 
 use ShopDB; 
 
--- Create tables 
-
-CREATE TABLE Products1 (
-    ID INT AUTO_INCREMENT,
-    Name VARCHAR(50),
-    PRIMARY KEY (ID)
-) ENGINE=InnoDB;
-
+-- Create table
 CREATE TABLE Products2 (
```

Comment:
The table name `Products2` might be a placeholder or a typo. If you intended to name it `Products`, consider renaming it for clarity.